_posts/2023-10-23-unidirectional-topic-operator.md

```diff
@@ -0,0 +1,311 @@
+---
+layout: post
+title: "Unidirectional Topic Operator"
```

Comment:
Maybe we can think of a better title? Not just the name of the operator you are going to talk about. Or even something like "Introduction to the new Unidirectional Topic Operator" or something better.

Reply:
There is no old Unidirectional TO. What about "Introducing the Unidirectional Topic Operator"?